app/src/main/res/values/colors.xml

```diff
@@ -2,7 +2,7 @@
 <resources>
     <color name="colorPrimary">#F08A5D</color>
     <color name="colorPrimaryDark">#000000</color>
-    <color name="colorAccent">#F08A5D</color>
+    <color name="colorAccent">#f08a5d</color>
```

Comment:
why some colors use all capital letters and some not. Make them all look consistent, please